pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/impl/BloomFilterRule.java

```diff
@@ -24,11 +24,11 @@
 import org.apache.pinot.controller.recommender.rules.AbstractRule;
 import org.apache.pinot.controller.recommender.rules.io.params.BloomFilterRuleParams;
 import org.apache.pinot.controller.recommender.rules.utils.FixedLenBitset;
-import org.apache.pinot.core.query.request.context.ExpressionContext;
-import org.apache.pinot.core.query.request.context.FilterContext;
 import org.apache.pinot.core.query.request.context.QueryContext;
-import org.apache.pinot.core.query.request.context.predicate.Predicate;
 import org.apache.pinot.core.requesthandler.BrokerRequestOptimizer;
+import org.apache.pinot.segment.spi.misc.ExpressionContext;
+import org.apache.pinot.query.spi.FilterContext;
+import org.apache.pinot.segment.spi.misc.Predicate;
```

Comment:
Why are we moving these 3 classes into different packages? They should be under the query-spi if you need to move them

Reply:
Sorry, based on comments I was removing the `pinot-query-spi` module I had added to park these files. Somehow, IDE did not fix the move correctly. It is fixed now.